client/my-sites/stats/sections/all-time-views-section/style.scss

```
@@ -1,6 +1,27 @@
 @import "calypso/my-sites/stats/modernized-stats-table-styles";
 
+.stats__all-time-views-section {
+	.highlight-cards-list {
+		display: block;
+	}
+
+	.highlight-card-heading {
+		flex-wrap: wrap;
+		gap: 16px
+	}
+
+	.highlight-card-heading > * {
+		flex: 1;
```

```suggestion
		flex: auto;
```
`min-width: fit-content` can be replaced with `flex-basis: auto`.